lib/ClimaLandSimulations/src/utilities/inputs_dataframe.jl

```diff
@@ -0,0 +1,112 @@
+export make_inputs_df
+
+"""
+    make_inputs_df(LOCAL_DATETIME, drivers)
+
+Returns DataFrames of inputs, one unitless, one in SI units, one in commonly used units. 
+"""
+function make_inputs_df(LOCAL_DATETIME, drivers)
+
+    variables_name = (
+        "DateTime",
+        "RECO",
+        "TA",
+        "VPD",
+        "PA",
+        "P",
+        "WS",
+        "LW_IN",
+        "SW_IN",
+        "CO2",
+        "SWC",
+        "TS",
+        "GPP",
+        "LE",
+        "H",
+        "G",
+        "SW_OUT",
+    ) # I imagine this may not work for all fluxnet sites...
```

Comment:
What happens if they dont exist?

Note that data_tools and met_drivers_FLUXNET already specify units for these fields, and specify which ones must be present to run the simulation. it seems like maybe this should rely on the output of the functions in those files? or maybe this should go in data_tools? Im not sure.

What is the dataframe used for? is it possible to use code already in met_drivers_FLUXNET to set up the dataframe?

Reply:
I think the difference here is using Unitful to make unit conversion easy. 
I use the DataFrame for the plots. 
With users in mind, using Unitful and DataFrame is a good idea in my opinion, but I kept the met_drivers_FLUXNET as we are currently using that format in fluxnet_simulation. I could refactor the whole thing though, I agree it is a bit confusing to generate multiple format of the same thing